text/0060-tasks-api.md

```diff
@@ -568,6 +568,137 @@ New task types are also added for these operations. `indexCreation`, `indexUpdat
 }
 ```
 
+#### 9. Paginate `task` resource lists
+
+The API endpoints `/tasks` and `indexes/{indexUid}/tasks` are browsable using a keyset based pagination.
+
+##### 9.1 Why a keyset based pagination?
+
+Keyset-based pagination is more appropriate when the data can grow or shrink quickly in terms of magnitude.
+
+###### 9.1.1 Pros
+
+The performance is better than the not-so-good but old pagination with `offset`/`limit`.
+
+Cursor pagination keeps the results consistent between each page as the data evolves. It avoids the [Page Drift effect](https://use-the-index-luke.com/sql/partial-results/fetch-next-page), especially when the data is sorted from the most recent to the oldest.
```

Comment:
Is cursor still the term you want to use here?

Reply:
No! Good spot!